website/src/theme/DocItem/index.js

```diff
@@ -1,115 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-import React from 'react';
-import DocPaginator from '@theme/DocPaginator';
-import DocVersionBanner from '@theme/DocVersionBanner';
-import Seo from '@theme/Seo';
-import LastUpdated from '@theme/LastUpdated';
-import TOC from '@theme/TOC';
-import EditThisPage from '@theme/EditThisPage';
-import {MainHeading} from '@theme/Heading';
-import clsx from 'clsx';
-import styles from './styles.module.css';
-import {
-  useActivePlugin,
-  useActiveVersion,
-  useVersions,
-} from '@theme/hooks/useDocs';
-
-import DocsRating from '../../../core/DocsRating';
-
-function DocItem(props) {
-  const {content: DocContent, versionMetadata} = props;
-  const {metadata, frontMatter} = DocContent;
-  const {
-    image,
-    keywords,
-    hide_title: hideTitle,
-    hide_table_of_contents: hideTableOfContents,
-  } = frontMatter;
-  const {
-    description,
-    title,
-    editUrl,
-    lastUpdatedAt,
-    formattedLastUpdatedAt,
-    lastUpdatedBy,
-    unversionedId,
-  } = metadata;
-  const {pluginId} = useActivePlugin({
-    failfast: true,
-  });
-  const versions = useVersions(pluginId);
-  const activeVersion = useActiveVersion(pluginId);
-  const showVersionBadge = versions.length > 1 && !activeVersion.isLast;
```

Comment:
`showVersionBadge` line was also altered to hide badge in the current docs in comparison to the unswizzled version.

Reply:
fixed with a new option